app/src/components/UploadImage/UploadImageButton.tsx

```diff
@@ -33,17 +46,131 @@ function UploadImageButton() {
     //     }
     // }
 
-    return (
-        <>
+    const addMoreButtons = () => {
+        return (
+            <>
+                <Button
+                    component="label"
+                    style={{
+                        position: 'absolute',
+                        color: '#55B1EB',
+                        backgroundColor: 'transparent',
+                    }}
+                >
+                    <div
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            alignItems: 'center',
+                        }}
+                    >
+                        <AddCircleOutlineIcon />
+                        <Typography>Add more Images</Typography>
+                        <input
+                            type="file"
+                            accept="image/*"
+                            multiple
+                            onChange={imageSelectHandler}
+                            style={{ display: 'none' }}
+                        />
+                    </div>
+                </Button>
+            </>
+        )
+    }
+
+    let addMoreImages: string | JSX.Element =
+        selectedImage.length > 0 ? (
             <Box
-                // boxShadow={3}
-                // display="flex"
-                // justifyContent="center"
-                width="75%"
-                margin="0 auto"
-                // border={3}
-                mt={5}
+                display="flex"
+                justifyContent="space-evenly"
+                style={{ margin: '2rem 5.125rem 0 5.125rem' }}
+            >
+                {[1, 2, 3].map((i) => {
+                    return (
+                        <div style={ContainerStyles}>
+                            {selectedImage[i] ? (
+                                <div style={{ width: '100%', height: '100%' }}>
+                                    <img
+                                        src={selectedImage[i]}
+                                        width="100%"
+                                        height="100%"
+                                    />
+                                </div>
+                            ) : (
```

Comment:
rather than invoking `addMoreButtons()` call it as a JSX component (should be renamed to a capitalized component name, e.g. `<AddMoreButtons />`

Reply:
just resolved this issue